client/js/utils/dom-events.js

```diff
@@ -1,10 +1,12 @@
 import fromEvent from 'xstream/extra/fromEvent';
 import debounce from 'xstream/extra/debounce';
 import throttle from 'xstream/extra/throttle';
+import xs from 'xstream';
 
 export const onWindowResizeDebounce$ = fromEvent(window, 'resize').compose(debounce(500));
 export const onWindowScrollThrottle$ = fromEvent(window, 'scroll').compose(throttle(100));
 export const onWindowScrollDebounce$ = fromEvent(window, 'scroll').compose(debounce(500));
+export const onWindowScrollThrottleDebounce$ = xs.merge(onWindowScrollThrottle$, onWindowScrollDebounce$);
```

Comment:
[Docs](https://github.com/staltz/xstream/wiki/Migrating-from-RxJS#from-rxjs-to-xstream) suggest `Observable.merge(a$, b$, c$)` in RxJS is written as `xs.merge(a$, b$, c$)` in xstream with no mention of chainability as far as I can tell.

As before, I want a throttled event that is also guaranteed to fire at the end – something xstream's `throttle` doesn't give us out of the box